EIPS/eip-725.md

```diff
@@ -1,55 +1,93 @@
----
+```
 eip: 725
-title: Proxy Account
-author: Fabian Vogelsteller (@frozeman), Tyler Yasaka (@tyleryasaka)
+title: ERC-725 Smart Contract Based Account
+author: Fabian Vogelsteller <fabian@lukso.network>, Tyler Yasaka (@tyleryasaka)
 discussions-to: https://github.com/ethereum/EIPs/issues/725
 status: Draft
 type: Standards Track
 category: ERC
+requires: ERC165, ERC173, ERC1271 (optional)
 created: 2017-10-02
----
+updated: 2020-07-02
+```
+
+**This is the new 725 v2 standard, that is radically different from [ERC 725 v1](https://github.com/ethereum/EIPs/blob/ede8c26a77eb1ac8fa2d01d8743a8cf259d8d45b/EIPS/eip-725.md). ERC 725 v1 is be moved to #734 as a new key manager standard.**
+--------
```

Comment:
```suggestion
```
EIPs should be written for future readers who don't care about the history.  If someone wants to see the progression of an EIP, they can go read through git history or the discussion link.

Reply:
removed